tests/typescript.test.mjs

```diff
@@ -0,0 +1,187 @@
+import { describe, it, beforeAll, afterAll, expect } from 'vitest';
+import { join } from 'path';
+import tmp from 'tmp-promise';
+import { execa } from 'execa';
+import copyWithTemplate from '../lib/copy-with-template';
+import { existsSync, writeFileSync } from 'fs';
+import stripAnsi from 'strip-ansi';
+
+const blueprintPath = join(__dirname, '..');
+const appName = 'fancy-app-in-test';
+
+describe('typescript', function () {
+  let tmpDir;
+  let emberCli = `ember-cli@5.11.0`;
```

Comment:
same as above, I don't want ember-cli version specified in the test. It should be driven by the package.json